### PR TITLE
Add message author check

### PR DESCRIPTION
**Pull request by micschwarz**
### Changelog
- Add message author check

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ client.on('ready', () => {
 });
 
 client.on('message', (msg) => {
+    // If author is bot exit
```
Comment (jens1o) [inline src/index.ts]: `do not care about the message if the author is a bot`

**Comment by jens1o** [comment]
// push

**Comment by micschwarz** [comment]
Abnehmen bitte, ist schon ewig fertig

**Comment by jens1o** [comment]
@micschwarz 

English please? ;) And I'd like the comment to change.

**Pull request merged**